azcli_conformity.sh

```diff
@@ -0,0 +1,60 @@
+# Define Variables for the Configurations
+export APP_SECRET=$(uuidgen)
+
+# Ask User Input for the App Registration Name
+echo " "
+echo "Which name do you want to give to your App Registration?"
+IFS=$'\n'
+read -r APP_NAME
+echo " "
+echo "Hello, Thank you for trying this script out.
+We're configuring your Azure Account to work with Cloud One Conformity, this might take several minutes depending of how many subscriptions you have."
+
+# Create App Registration
+az ad app create --display-name "$APP_NAME" --password $APP_SECRET --credential-description secret --required-resource-accesses @manifest.json > /dev/null
+
+# Export App Registration ID and Active Directory ID
+export AD_ID=`az account show --query tenantId --output tsv`
```

Comment:
❓ What is the value in `export`ing all of these variables?

Reply:
We need these variables to print at the end of the script so customers using the script can finish the integration in the Conformity console. As soon as the Azure API is released, we don't necessarily need to print these and we can just finish the integration on behalf of the customer by adding the Subscriptions via API.